README.md

```diff
@@ -18,24 +18,31 @@ Install the `RectorLaravel` package as dependency:
 composer require driftingly/rector-laravel --dev
 ```
 
-## Use Sets
+## Automate Laravel Upgrades
 
-To add a set to your config, use `RectorLaravel\Set\LaravelSetList` class and pick one of the constants:
+To add a set to your config, use `RectorLaravel\Set\LaravelLevelSetList` and pick the constant that matches your target version.
+Sets for higher versions include sets for lower versions.
 
 ```php
-<?php
+return Rector\Config\RectorConfig::configure()
+    ->withSets([
+        RectorLaravel\Set\LaravelLevelSetList::LARAVEL_110,
+    ]);
+```
 
-declare(strict_types=1);
+## Additional Sets
 
-use Rector\Config\RectorConfig;
-use RectorLaravel\Set\LaravelSetList;
+The other sets do not correlate directly to Laravel upgrades, but can be used to improve different aspects of your code.
+Use `RectorLaravel\Set\LaravelSetList` to add sets.
 
-return RectorConfig::configure()
+```php
+return Rector\Config\RectorConfig::configure()
     ->withSets([
-        LaravelSetList::LARAVEL_110
+        RectorLaravel\Set\LaravelSetList::LARAVEL_CODE_QUALITY,
+        RectorLaravel\Set\LaravelSetList::LARAVEL_COLLECTION,
```

Comment:
Please use the same format as the existing version, with `declare` and imports:
```php
<?php

declare(strict_types=1);

use Rector\Config\RectorConfig;
use RectorLaravel\Set\LaravelSetList;

return RectorConfig::configure()
```
I think we can add here `LaravelSetList::LARAVEL_110`, on top of code quality and collection sets, to show that it exists apart from the `UP_TO_` sets.

Reply:
I added an explanation in the previous section `Automate Laravel Upgrades`.